go/image.bzl

```diff
@@ -89,7 +89,7 @@ STATIC_DEFAULT_BASE = select({
     "//conditions:default": "@go_image_static//image",
 })
 
-def go_image(name, base = None, deps = [], layers = [], env = {}, binary = None, **kwargs):
+def go_image(name, base = None, deps = [], layers = [], env = {}, binary = None, architecture = "x86_64", **kwargs):
```

Comment:
The existing default is amd64 (set in container/image.bzl). Rather than setting a new default here and changing the behavior for existing users, set architecture = None

Reply:
Good point, changed for all three.